service/matching/backlog_manager.go

```diff
@@ -160,6 +159,7 @@ func (c *backlogManagerImpl) Stop() {
 		defer cancel()
 
 		_ = c.db.UpdateState(ctx, ackLevel)
+		// TODO: Says we should ignore any errors, but can an error occur when UpdatingState?
```

Comment:
was just thinking along the lines of "what happens if we can't update state in the very last time?" Do we take care of this or let it be?

Reply:
This shall be marked as resolved due to the comment which cleared the question: 

	_// Note that it's fine to GC even if the update ack level fails because we did match the
	// tasks, the next owner will just read over an empty range._